service/controller/aws/v10/version_bundle.go

```diff
@@ -12,11 +12,16 @@ func VersionBundle() versionbundle.Bundle {
 				Description: "Added cluster-autoscaler chartconfig.",
 				Kind:        versionbundle.KindAdded,
 			},
+			{
+				Component:   "kube-state-metrics",
+				Description: "Updated to 1.5.0. More info here: https://github.com/kubernetes/kube-state-metrics/blob/v1.5.0/CHANGELOG.md",
```

Comment:
Do we usually link to the changelog? I like the idea but if this is something we want to do we should also update `fmt`.

For App Catalog it would be great to link to the changelog from Happa. We could have that as an annotation in the Chart.yaml.

Reply:
I saw we started to do latest release
```
Changelog:
  aws-operator: Fixed a bug with removing CR finalizers before all Cloud Formation stacks are deleted.
  kubernetes: Updated Kubernetes to 1.12.1. More info here: https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.12.md
  calico: Updated to 3.2.3. Also the manifest has proper resource limits to get QoS policy guaranteed.
  kubernetes: Enabled admission plugins: DefaultTolerationSeconds, MutatingAdmissionWebhook, ValidatingAdmissionWebhook.
  container-linux: Updated to latest stable 1855.5.0
  etcd: Updated to 3.3.9
  docker: Updated to 18.06.1
  vault: Vault version updated.
  chart-operator: Add your changes here.
  cluster-operator: Allow advanced CoreDNS configuration. See https://docs.giantswarm.io/guides/advanced-coredns-configuration
```

Reply:
OK cool! I like the idea.

Reply:
I did mention that in some fmt discussion IRRC, we should always include a link to upstream changelog